testt.py

```diff
@@ -0,0 +1,19 @@
+#a=9
+#print("a is",a)
+
+import shutil
+
+class Sam:
+    def __init__(self):
+        pass
+
+
+    def mainp(self):
+        src = "testt"  # your base file
+
+        des = '/Users/b.s.v.prasad/Desktop/'  # destination Filelocation
```

Comment:
give path as ~/Desktop/uploads... 
u need to ensure such a folder exist if it does not u need to create one.